### ISSUE TITLE
zsh: no matches found: camelot-py[plot]

### ISSUE THREAD

**Issue by jorgesisco** [camelot-dev/camelot#306]
I am trying to install plot `pip install camelot-py[plot]` to perform some visual debugging and I get the message shown on this issue's title.

am I missing something here?

**Comment by huyz** [comment]
Yes, you're probably using a shell such as zsh that interprets `[` and `]` as special for file globbing.
So just do:

`pip install 'camelot-py[plot]'`

**Comment by DoomedJupiter** [comment]
closing as resolved
